deploy/olm-catalog/ocs-operator/0.0.1/role.yaml

```diff
@@ -0,0 +1,16 @@
+kind: Role
```

Comment:
We need to ensure that the unified csv generation is creating these files. If not we'll miss out on these when a new version of the csv is generated.
@davidvossel can you help @umangachapagain here?

Reply:
rook/ceph/monitoring has these rbacs if you wish to automate. But, for now I would suggest to Keep It Simple.